src/main.js

```diff
@@ -2,6 +2,10 @@
 
 // eslint-disable-next-line import/no-extraneous-dependencies, import/extensions
 import { CompositeDisposable } from 'atom'
+import store from './store/main'
+import { getFixableAsIgnored } from './store/selectors'
+// eslint-disable-next-line import/prefer-default-export
+export { store }
```

Comment:
If you are referring to the fact that I'm both importing and exporting `store`, it because it is essentially an instance, so I need to send a reference to that same instance to the helpers. If we were merging this, I could probably make it more clear by _not_ calling `createStore` automatically on the export default this is importing from.  So then I would need to call `createStore()` here and it would be more clear that it was an instance. With a little work and refactoring, the eventual goal would have been that any functions in helpers that need to know about the state or call dispatch would be passed the state or the store as a parameter.